### PR TITLE
RDoc-1649: added new pages how-to-filter-by-field and check-if-attachment-exists

### PR DESCRIPTION
**Pull request by morhilai**
-made c# and java versions
-linked to how-to-filter-by-field.markdown from what-is-rql.markdown and what-is-document-query.markdown
-Improved existing page check-if-document-exists.markdown

### CODE REVIEW

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/querying/.docs.json [hunk 1]
```diff
@@ -86,6 +86,12 @@
     "DiscussionId": "6c7d2633-e0ec-4b8f-8ddd-43470f24aa29",
     "Mappings": []
   },
+  {
+    "Path": "how-to-filter-by-field.markdown",
+    "Name": "How to Filter by Field",
```
Comment (maximburyak) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/querying/.docs.json]: I think that this title might be considered ambiguous.
Maybe we should modify that to `How to Filter by Field Presence` or something better

Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/querying/.docs.json]: How about `How to Filter by Existing Fields`?
If that's good - fixed

Comment (maximburyak) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/querying/.docs.json]: It still misses the specificity of filtering by field presence and not values. @Danielle9897 what is your opinion regarding that?

Comment (Danielle9897) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/querying/.docs.json]: => How to filter by fields existence

**Pull request merged**